### DISCUSSION TITLE
Can we create a label for fixes which newbie can fix ?

### DISCUSSION THREAD

**Discussion by VikalpRusia** [rancher-sandbox/rancher-desktop#7536] [Ideas]
Hi Team,
I want to contribute (newcomer) to this project but there is no label for starting contributors.

Ask
1. can we have a label for some features/bug which can be picked by newcomer.
2. can you please post a bug which I can pick (newcomer) here

**Comment by jandubois** [comment]
Thank you for your interest! It would be helpful to know which area you would like to contribute to, and what your existing skillset is. E.g. if you have experience with TypeScript and Electron, then we could look for a GUI issue. But if you are more familiar with Go code, we would look for something in the backend supported utilities, etc.

**Reply by VikalpRusia** [reply]
I have experience in Typescript and willing to learn the Electron/Go language so both works.

**Reply by jandubois** [reply]
If you want to start with something very easy then you may want to look into https://github.com/rancher-sandbox/rancher-desktop/issues/7508. I've closed it because I don't think the bug is very important. It would just require putting the HTML-escaped string into the message instead of letting `clean-html` just strip it out.

https://github.com/rancher-sandbox/rancher-desktop/issues/6733 is another snapshot UI issue that hopefully has a simple fix (only set the timestamp once, don't update it when the page is refreshed).

**Reply by VikalpRusia** [reply]
sure will pick this up one quick question while setting up it mentions specifically mentions - https://github.com/rancher-sandbox/rancher-desktop?tab=readme-ov-file#%EF%B8%8F-working-on-a-mac-with-an-m1-chip

I have M3 do I need to take care of something extra?

**Reply by jandubois** [reply]
Yes, you should `export M1=1` to make sure you build/download aarch64 binaries instead of x86_64. You need to do this before you run `yarn` because it will fetch various bits as part of the `postinstall` action, and needs the correct architecture.

**Reply by VikalpRusia** [reply]
@jandubois are there more bugs which i can pick?

**Reply by jandubois** [reply]
I've assigned #7559 (the one you just filed yourself) to you.

If you have an external monitor on your machine, then #7318 would be something we are interested in fixing.

Another option would be #6514.

I'll keep looking for more.

Let me know if I should assign any of these to you as well!

**Reply by VikalpRusia** [reply]
Hi @jandubois #7318 seems to be working good on my MAC and others I have raised MR for.
Are there more bugs which I can pick?

**Reply by jandubois** [reply]
I'll take a look; I think #5582 would be a good issue. Make sure to use an existing library to convert a timestamp into something like "2 days ago" and don't implement it yourself. Maybe https://day.js.org/docs/en/plugin/relative-time, but I haven't looked if there is something more popular.

**Reply by jandubois** [reply]
Hi @VikalpRusia, haven't seen you in a bit. If you are still interested, then #7516 would be another issue that should be relatively easy to implement.

**Reply by VikalpRusia** [reply]
sure I can pick it, went on vacations :)

**Reply by jandubois** [reply]
Yeah, no rush! Enjoy your vacation.

But when you are ready, there is also some feedback from @mook-as and me on #7576 for you to look at.

**Reply by VikalpRusia** [reply]
I am back now will address comment and pick it

**Reply by VikalpRusia** [reply]
Sorry for long long delay got occupied in some family stuff, I will restart the work on #75756.
